verse/analysis/verifier.py

```diff
@@ -523,6 +526,10 @@ def compute_full_reachtube(
             if len(self.verification_queue) > 0:
                 # print([node.id for node in verification_queue])
                 node, later = self.verification_queue.pop(0)
+                # check height
+                if node.height >= max_height-1:
+                    print("max depth reached")
+                    break
```

Comment:
Let's use continous instead of break, because we want to get everything at max_height instead of stop.